src/style.css

```diff
@@ -1,3 +1,54 @@
 body {
   margin: 0;
 }
+
+.header{
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.header img{
+  margin: 10px 0;
+}
+
+.nav_list{
+  display: flex;
+  list-style: none;
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Style all elements using `.class` and if needed with `:pseudo-class`, `pseudo-element` and `[attribute]`. Here you're styling `ul` directly, consider adding a class to the `ul` and style that class instead.